app/src/main/res/values/strings.xml

```diff
@@ -197,11 +197,17 @@ Tap this message (or hit back) to skip this step.</string>
   <string name="navigation_item_feedback">Feedback</string>
   <string name="navigation_item_logout">Logout</string>
   <string name="navigation_item_info">Tutorial</string>
+  <string name="navigation_item_developer_plans">Developer plans</string>
 
   <string name="nearby_needs_permissions">Nearby places cannot be displayed without location permissions</string>
   <string name="no_description_found">no description found</string>
   <string name="nearby_info_menu_commons_article">Commons file page</string>
   <string name="nearby_info_menu_wikidata_article">Wikidata item</string>
-    <string name="error_while_cache">Error while caching pictures</string>
+  <string name="error_while_cache">Error while caching pictures</string>
+  <string name="feedback_popup_title">Feedback wanted</string>
+  <string name="feedback_popup_description">We are planning several new features and improvements for the app! Would you like to review them and provide feedback? (You can always access this by selecting "Developer plans" in the navigation drawer)</string>
```

Comment:
Minor issue - it looks to me like it might be good to have a line break between `We are planning several new features and improvements for the app! Would you like to review them and provide feedback? `and `(You can always access this by selecting "Developer plans" in the navigation drawer)`. I didn't realize this before I tried it, but it's a bit wall-of-text-ish without a line break.